modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/display/view.jsp

```diff
@@ -299,8 +300,14 @@ long formInstanceId = ddmFormDisplayContext.getFormInstanceId();
 						</c:choose>
 					}
 
+					<c:choose>
+						<c:when test="<%= autoRememberMe %>">
+							var autoRememberMe = true;
+						</c:when>
+					</c:choose>
+
 					<portlet:namespace />sessionIntervalId = setInterval(function () {
-						if (Liferay.Session) {
+						if (Liferay.Session || autoRememberMe) {
```

Comment:
@joseabelenda, after analyzing you changes on `view.jsp`, I feel that all of them can be summarized on a single line, e.g. invoke `ddmFormDisplayContext.isAutoRememberMe()` here, what do you think?

This way I believe we'll save some lines. 🙂

Reply:
@natocesarrego, I prefer use separated blocks (Java e Javascript) for better readability.
If you prefer, feel free to proceed. 🙂